ingress/ingress.go

```diff
@@ -28,7 +28,6 @@ var (
 )
 
 const (
-	ServiceBastion     = "bastion"
```

Comment:
Consolidate the use via `config.BastionFlag` and also to avoid `import cycle not allowed in test` which got raised when I was referencing `ingress` from `carrier`